QUARKUS-1325.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-1325 - Helm chart support for Quarkus
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1325
+
+This test plan cover Quarkus - Helm integration
+
+## Scope of the testing
+
+Verify a repeatable/versioned way to deploy Quarkus application into K8s/ocp through helm
+
+Test development will focus on
+- Generate and deploy a Quarkus Helm Chart over Openshift default repository
+- Verify some mapping between Quarkus configuration and generated Helm chart:
+  - Quarkus SmallRye Health 
+
+### Impact on testsuites and testing automation:
+  - Github CI jobs will be excluded from Helm verification. 
+  - Jenkins slaves instances will have Helm / helmfile installed and will deploy a Quarkus application through Helm into an Openshift cluster instance. This is a new way to deploy into K8s/Openshift that will increase slightly the total execution of time of Quarkus Test-suite.  
+  - Charts should be removed from Openshift default repository after test execution 
+  
+## Getting familiar with the feature
+Following actions were taken to ensure familiarity:
+ - Exploratory testing with current Quarkiverse extension
+ - Ensure good user experience and simplicity of use
+
```

Comment:
I would add something like "Ensure upstream CI coverage" to check if the extension has CI against Kubernetes.